backend/test/test_collections.py

```diff
@@ -382,6 +382,9 @@ def test_get_collection_replay(crawler_auth_headers, default_org_id):
     assert data["dateEarliest"]
     assert data["dateLatest"]
     assert data["defaultThumbnailName"]
+    assert data["pagesQuery"].endswith(
```

Comment:
Also want check that `pages`/`seedPages` are returned here and in the test below

Reply:
Added!